pmm-app/src/pmm-add-instance-app-panel/AddInstance/AddRemoteInstance/AddRemoteInstance.tsx

```diff
@@ -66,6 +66,20 @@ const getAdditionalOptions = (type, form) => {
         </>
       );
     case 'MySQL':
+      if (remoteInstanceCredentials.isRDS) {
```

Comment:
What do you think about this? @lunaticusgreen 
```suggestion
      return (
          <>
            <CheckboxField form={form} label={'Use performance schema'} name="qan_mysql_perfschema" />
            <span className="description"></span>

      (remoteInstanceCredentials.isRDS ? (
            <CheckboxField form={form} label={'Disable Basic Metrics'} name="disable_basic_metrics" />
            <span className="description"></span>

            <CheckboxField form={form} label={'Disable Enhanced Metrics'} name="disable_enhanced_metrics" />
            <span className="description"></span>
            ) : (Null))
          </>
        );
```

Reply:
@BupycHuk 
There is no such thing as `(Null)` in js. What dit you mean?

Reply:
It looks ok, but instead of `Null` you have to use `null`

Reply:
null